service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java

```diff
@@ -183,26 +183,15 @@ public void run() {
   private void loadFrameworkVersions() {
     Framework framework = new Framework();
     framework.setName(CONFIG_FRAMEWORK_DEFAULT_NAME);
-
-    if (needSetFrameworkVersion()) {
-      framework.setVersion(FrameworkVersions.allVersions());
+    if(ServiceRegistryConfig.getMicroserviceFramework()) {
```

Comment:
1.seems that Version and VersionUtils import but not used
2.when will not report framework version?

Reply:
1.yes, i will modify it
2.when use old Service Center version like 2.2.49, Service Center is not support Framework json object like this xxx:xxx

Reply:
if want to be compatible with old SC, then judge by serviceCenterInfo if better
but it's not a good idea to get serviceCenterInfo  from SC when serviceRegistry init, because if failed to get, will cause problem
it's better go get serviceCenterInfo  in service center task

Reply:
I think maybe I am not describe this problem clearly 。what I think is  provide a switch so when using old SC ,configuring "cse.service.registry.framework = false" and then SC validate will through. at last ,service will registry successfully.